app/_services/dates.ts

```diff
@@ -1,9 +1,5 @@
-import dayjs from 'dayjs';
-import bigIntSupport from 'dayjs/plugin/bigIntSupport';
-import localizedFormat from 'dayjs/plugin/localizedFormat';
-dayjs.extend(bigIntSupport);
-dayjs.extend(localizedFormat);
-
-export function formatTimestampSecondsAsLocal(timestampSeconds: bigint) {
-	return dayjs(timestampSeconds * BigInt('1000')).format('L LT');
+export function formatTimestampSecondsAsLocal(timestampSeconds: bigint, locale?: string) {
+	const selectedLocale =
+		locale || (typeof navigator !== 'undefined' ? navigator.language : 'en-US');
```

Comment:
i don't think we should default to US times because the month and day are swapped relative to the rest of the world, `en-gb` would make more sense